curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff855e57d1b220fd53b4b0.md

```diff
@@ -0,0 +1,148 @@
+---
+id: 63ff855e57d1b220fd53b4b0
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Now, it's time to pass a callback to the `forEach` so we can start populating the author items we destructured. We will put all of the items inside our `authorContainer` `div` by using the `innerHTML` `DOM` method.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here